client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -60,6 +60,70 @@ function PlaceholderPostEdit( props ) {
 	);
 }
 
+function RelatedPostsEditItem( props ) {
+	const previewClassName = 'related-posts__preview';
+
+	return (
+		<div className={ `${ previewClassName }-post` }>
+			{ props.displayThumbnails && props.post.img && props.post.img.src && (
+				<Button className={ `${ previewClassName }-post-link` } isLink>
+					<img src={ props.post.img.src } alt={ props.post.title } />
+				</Button>
+			) }
+			<h4>
+				<Button className={ `${ previewClassName }-post-link` } isLink>
+					{ props.post.title }
+				</Button>
+			</h4>
+			{ props.displayDate && (
+				<span className={ `${ previewClassName }-post-date has-small-font-size` }>
+					{ props.post.date }
+				</span>
+			) }
+			{ props.displayContext && (
+				<p className={ `${ previewClassName }-post-context` }>{ props.post.context }</p>
+			) }
+		</div>
+	);
+}
+
+function RelatedPostsPreviewRows( props ) {
+	const className = 'related-posts__row';
+
+	let topRowEnd = [];
+	let bottomRowStart = [];
```

Comment:
These are set to integer values further down -- ideally don't init them as (empty) arrays since that's a different data type altogether, which is something i personally stumble over a bit when reading the code. 

I'd suggest either leaving it uninitialized, or setting it to what further down is used as the `default:` value in the `switch`.

Reply:
Yes. This is silly. I'll fix it right away.